superset-frontend/package.json

```diff
@@ -121,6 +121,7 @@
     "abortcontroller-polyfill": "^1.1.9",
     "ace-builds": "^1.4.14",
     "antd": "4.10.3",
+    "antd-v5": "npm:antd@^5.18.0",
```

Comment:
The previous and target versions of Ant Design can be supported by using different names in the package configuration.